src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/slurm/SlurmCommandParser.groovy

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2017 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.roddy.execution.jobs.cluster.slurm
+
+
+import de.dkfz.roddy.config.ResourceSet
+import de.dkfz.roddy.execution.jobs.BEJobID
+import de.dkfz.roddy.execution.jobs.GenericJobInfo
+import de.dkfz.roddy.tools.BufferUnit
+import de.dkfz.roddy.tools.BufferValue
+import de.dkfz.roddy.tools.ComplexLine
+import de.dkfz.roddy.tools.TimeUnit
+import groovy.transform.CompileStatic
+
+import static de.dkfz.roddy.StringConstants.SPLIT_COMMA
+import static de.dkfz.roddy.StringConstants.SPLIT_EQUALS
+
+/**
+ * Used to convert commands from cli to e.g. GenericJobInfo
+ *
+ * TODO It needs further work, in case it is needed
+ */
+@CompileStatic
+class SlurmCommandParser {
+
+    String[] options
+    String jobName
+    String walltime
+    String memory
+    BufferUnit bufferUnit = BufferUnit.G
+    String cores
+    String nodes
+    String queue
+    String otherSettings
+    String script
+    BEJobID jobID
+    Map<String, String> parameters = new LinkedHashMap<>()
+    List<String> dependencies = new LinkedList<>()
+
+    String commandString
+
+    SlurmCommandParser(String commandString) {
+        this.commandString = commandString
+        parse()
+    }
+
+    void parse() {
+        commandString = commandString.trim()
+        // Get the job id
+        jobID = new BEJobID(commandString.substring(0, commandString.indexOf(",")))
+        // Get rid of the job id
+        commandString = commandString.substring(commandString.indexOf(",") + 1).trim()
+
+        // Create a complex line object which will be used for further parsing.
+        ComplexLine line = new ComplexLine(commandString)
+
+        if (!commandString.startsWith("qsub")) return  // It is obviously not a PBS call

```

Comment:
I know, its also not adapted in the LSFCommandParser, but here -- with the same command `qsub` as in PBS -- it is really confusing to say "PBS call" in the comment.

```suggestion
        if (!commandString.startsWith("qsub")) return  // It is obviously not a SLURM call

```